app/views/comments/_comments.html.slim

```diff
@@ -0,0 +1,10 @@
+- content_for :javascript do
```

Comment:
IN EVERY partial you are including this JS?
include it in application.js
and just initialize on comments page (call some JS).